kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java

```diff
@@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
         session.setUserId("admin");
 
         _ss = (SiteService)getService(SiteService.class.getName());
-        Site newSite = _ss.addSite(SITE_ID, (String) null);
-        Group group = newSite.addGroup();
-        group.setTitle(".group");
-        _groupReference = group.getReference();
-        _ss.save(newSite);
+        Site newSite = null;
```

Comment:
Unless I'm missing something, I'm not sure how can I remove the site like
it used to, considering the property change associated is what's postponing
the sites deletion and causing the tests issues. Same for Juanjo's comment,
please let me know how could I delete the sites on the test files and I
will. Thanks.

2016-06-27 16:18 GMT+02:00 Matthew Buckett <notifications@github.com>:

> In
> kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java
> <https://github.com/sakaiproject/sakai/pull/2925#discussion_r68583809>:
>
> > @@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
> >          session.setUserId("admin");
> >
> >          _ss = (SiteService)getService(SiteService.class.getName());
> > -        Site newSite = _ss.addSite(SITE_ID, (String) null);
> > -        Group group = newSite.addGroup();
> > -        group.setTitle(".group");
> > -        _groupReference = group.getReference();
> > -        _ss.save(newSite);
> > +        Site newSite = null;
>
> Even if using the same site happens to work at the moment it makes the
> tests more brittle than they need to be, for example what happens if the
> test order is different after a jUnit upgrade. Why not just fix the test so
> it removes the site like it used to.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/sakaiproject/sakai/pull/2925/files/c348ce39511520ae4ce640595a668577db967d06#r68583809>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AK6IljeBnhWZPu8IGonbN8SJOatESOnlks5qP9uZgaJpZM4I9pie>
> .
>